flutter/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   sentry: ^6.5.0-alpha.1
   package_info_plus: ^1.0.0
   meta: ^1.3.0
+  stack_trace_parser: ^0.0.1
```

Comment:
@bruno-garcia what do you think about this? https://pub.dev/packages/stack_trace_parser
Created and maintained by @ueman but if it becomes popular, we might have dependency problems.
The other option is to vendor its code ^^

Reply:
I'm fine with both, just let me know.

Reply:
Another solution would be to move this into another library. `sentry-community` or something, which is allowed to make use of libraries and newer Dart/Flutter versions.

Reply:
I think it makes sense to inline the `stack_trace_parser` code. There are still quite a few subtle differences between the results I'm currently getting and how it looks for the native Android reports. Fixing those might require changes to `stack_trace_parser`.

Reply:
I've vendored the code